WTWSMS/common/religion_modifiers/WtWSMS_religion_modifiers.txt

```diff
@@ -14,12 +14,12 @@ affected_by_pestilence = {
 	years = 5
 }
 
-fall_of_rome= {
+fall_of_rome_MA = {
```

Comment:
Maybe "fall_of_rome_moral_authority" would be more explicit? Just my preference though.

Reply:
ok

Reply:
no, wait. I forgot to edit this file

Reply:
Now really done